src/Sylius/Behat/Context/Ui/Admin/ManagingPaymentMethodsContext.php

```diff
@@ -301,6 +309,23 @@ public function thisPaymentMethodShouldBeDisabled()
     }
 
     /**
+     * @Given the payment method :paymentMethod should have instructions :instructions in :language
+     */
+    public function thePaymentMethodShouldHaveInstructionsIn(
+        PaymentMethodInterface $paymentMethod,
+        $instructions,
+        $language
+    ) {
+        $this->iWantToModifyAPaymentMethod($paymentMethod);
+
+        Assert::same(
+            $this->updatePage->getPaymentMethodInstructions($language),
+            $instructions,
+            'Instructions of payment method are invalid'
```

Comment:
I'm not sure whether it is helpful to see just "Instructions of payment method are invalid". With this message being left empty, we will see an exception in `thePaymentMethodShouldHaveInstructionsIn` method with message `should be X but is Y instead`, so we won't spend time on debugging it (same reason as for #5417).